node/impl/full/gas.go

```diff
@@ -297,6 +297,47 @@ func gasEstimateGasLimit(
 
 	ret := res.MsgRct.GasUsed
 
+	transitionalMulti := 1.0
+	// Overestimate gas around the upgrade
+	if ts.Height() <= build.UpgradeFVM1Height && (build.UpgradeFVM1Height-ts.Height() <= 20) {
```

Comment:
How did you pick 20 here?

Reply:
This gives us 10 minutes of overestimation, there are few reasons I went with 20 epochs.
I would be comfortable with 5 minutes, if we didn’t have tipset system, possible off-by-ones somewhere so on, and I feel like increasing it to 10min has a low cost.

Reply:
Seems reasonable -- i don't think there's a "correct" answer here.